README.md

```diff
@@ -49,6 +50,22 @@ This gem provides a command-line interface which can be run like so:
 For example, `erblint --lint-all --enable-all-linters` will run all available
 linters on all ERB files in the current directory or its descendants (`**/*.html{+*,}.erb`).
 
+## Enable or disable default linters
+`EnableDefaultLinters`: enables or disables default linters. Default linters are enabled by default.
+These linters are enabled by default:
```

Comment:
Some of these seem to be listed in the README as the main features, maybe we could list them as a table and whether they are a default linter or not (and link the title to its section if one exists). Do you think this would make it easier to read through the README?

Reply:
Great idea! Took some time, but I added a new `Linter` table that shows if it is a default linter, a description, if available, and links to the part of the readme if an explanation exists. Would definitely be easier to see all the linters now. 

![image](https://user-images.githubusercontent.com/25471753/73094658-0e663680-3eaf-11ea-8b18-03fc08c37b59.png)

Reply:
Super nice 🚀

Reply:
Omg this is amazing ❤️ thanks for putting in the time to do this @wildmaples